scripts/qa_crowbarsetup.sh

```diff
@@ -3101,6 +3101,7 @@ function reboot_controller_clusters()
     for cluster in data network services; do
         local clusternodes_var=$(echo clusternodes${cluster})
         for machine in ${!clusternodes_var}; do
+            wait_for 400 5 "ssh $m_hostname 'if `which drbd-overview &> /dev/null`; then ! drbd-overview | grep -q Inconsistent; fi'" "drbd devices to be consistent on node $m_hostname"
```

Comment:
Thanks, no big difference, but adapted to use it as I guess we'll have less chances to have an accidental match.